src/orderbook.js

```diff
@@ -828,6 +828,32 @@ OrderBook.prototype.isBalanceChangeNode = function(node) {
   return true;
 };
 
+OrderBook.prototype._canRunAutobridgeCalc = function(): boolean {
+  return !this._calculatorRunning && (Date.now() - this._lastCalcDuration >= 100);
```

Comment:
You're right, logic is incorrect. 
OK, I'll throw this out, this was just a precaution.